src/display/svg.js

```diff
@@ -321,91 +322,101 @@ var SVGExtraState = (function SVGExtraStateClosure() {
     this.maskId = '';
   }
 
-  SVGExtraState.prototype = {
-    clone: function SVGExtraState_clone() {
-      return Object.create(this);
-    },
-    setCurrentPoint: function SVGExtraState_setCurrentPoint(x, y) {
-      this.x = x;
-      this.y = y;
-    },
-  };
-  return SVGExtraState;
-})();
-
-SVGGraphics = (function SVGGraphicsClosure() {
-  function opListToTree(opList) {
-    var opTree = [];
-    var tmp = [];
-    var opListLen = opList.length;
-
-    for (var x = 0; x < opListLen; x++) {
-      if (opList[x].fn === 'save') {
-        opTree.push({ 'fnId': 92, 'fn': 'group', 'items': [], });
-        tmp.push(opTree);
-        opTree = opTree[opTree.length - 1].items;
-        continue;
-      }
+  clone() {
+    return Object.create(this);
+  }
 
-      if (opList[x].fn === 'restore') {
-        opTree = tmp.pop();
-      } else {
-        opTree.push(opList[x]);
-      }
-    }
-    return opTree;
+  setCurrentPoint(x, y) {
+    this.x = x;
+    this.y = y;
   }
+}
 
-  /**
-   * Formats float number.
-   * @param value {number} number to format.
-   * @returns {string}
-   */
-  function pf(value) {
-    if (Number.isInteger(value)) {
-      return value.toString();
+// eslint-disable-next-line no-inner-declarations
+function opListToTree(opList) {
+  let opTree = [];
+  const tmp = [];
+
+  for (const opListElement of opList) {
+    if (opListElement.fn === 'save') {
+      opTree.push({ 'fnId': 92, 'fn': 'group', 'items': [], });
```

Comment:
Yes, in fact that's part of the follow-up after this patch since it's already included in https://github.com/mozilla/pdf.js/pull/10475/files and the objective is to rebase that after this lands.